mkt/__init__.py

```diff
@@ -1,18 +1,19 @@
+# flake8: noqa
```

Comment:
Also, is there any hope of removing the `import *` from this file and fix all the imports that use it in zamboni ? If there is, maybe we should file a bug. No idea how big of a task it is though.